specs/player_spec.rb

```diff
@@ -0,0 +1,162 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+require 'pry'
+require_relative '../lib/player'
+
+
+# Get that nice colorized output
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'Player' do
+
+  describe "name" do
+
+    it 'gets a name' do
+      Scrabble::Player.new("Chihiro").must_be_kind_of Scrabble::Player
+      Scrabble::Player.new("Haku").name.must_be :==,"Haku"
+      Scrabble::Player.new().name.must_equal "No Face"
+    end
+  end
+
+  describe "plays" do
+
+    it 'returns an array of words' do
+      haku = Scrabble::Player.new("Haku")
+      haku.plays = ["Haku", "dust bunnies"]
+      haku.plays[0].must_be_kind_of String
+      haku.plays[1].must_be_kind_of String
+      haku.plays[0].wont_be_kind_of Integer
+
+      haku.plays[0].wont_be_nil
+
+      Scrabble::Player.new.plays.must_be_kind_of Array
+    end
+  end
+
+  describe 'play(word)' do
+    before do
+      @first_player = Scrabble::Player.new("Yubaba")
+    end
+    it 'adds input to plays array' do
+      @first_player.plays.must_be_empty #before any plays
+
+      @first_player.play("dragon")
+      @first_player.plays.must_include "dragon"
+      @first_player.plays.wont_be_nil
+      @first_player.plays.length.must_equal 1
+      #binding.pry
+    end
+
+    it 'returns false if player has already won' do
+
+      @first_player.total_score.wont_be:<,0
+
+      @first_player.play("radish")
+      @first_player.total_score.must_be:>,0
+
+
+      @first_player.total_score = 110
```

Comment:
Don't "hack" your tests, you shouldn't be able to directly set the player's score.  Instead play enough words that the player has won, and then you can test.